File: designPattern/Builder/java/Director.java
public class Director{
	Builder builder;

	public Director(Builder builder){
		this.builder = builder;
	}

	public void constructor(){
		builder.setTitle("Greeting");
		builder.setText("아침과 저녁에는");
		builder.setItem(new String[]{
			"안녕하세요",
			"좋은 아침입니다."
			});
		builder.setText("저녁에는");
		builder.setItem(new String[]{
			"안녕하세요",
			"안녕히 주무세요",
			"저녁 드셨나요?"
			} );
		builder.close();
	}
}

File: designPattern/Builder/python/Director.py
from Builder import Builder


class Director:

    def __init__(self, builder):
        self.builder = builder

    def constructor(self):
        self.builder.setTitle("Greet")
        self.builder.setText("아침에는")
        self.builder.setItems(["goodMorning", "좋은 아침입니다.", "좋아"])
        self.builder.setText("저녁에는")
        self.builder.setItems(["goodEvening", "멋진 저녁입니다.", "좋아"])
        self.builder.close()